trakt/trakt.go

```diff
@@ -211,48 +219,66 @@ func TokenRefreshHandler() {
 		case <-closer:
 			return
 		case <-ticker.C:
-			if time.Now().Unix() > int64(config.Get().TraktTokenExpiry)-int64(259200) {
-				req := reqapi.Request{
-					API:    reqapi.TraktAPI,
-					Method: "POST",
-					URL:    "oauth/token",
-					Header: http.Header{
-						"Content-type": []string{"application/json"},
-						"User-Agent":   []string{UserAgent},
-						"Cookie":       []string{Cookies},
-					},
-					Params: napping.Params{
-						"refresh_token": config.Get().TraktRefreshToken,
-						"client_id":     config.TraktWriteClientID,
-						"client_secret": config.TraktWriteClientSecret,
-						"redirect_uri":  "urn:ietf:wg:oauth:2.0:oob",
-						"grant_type":    "refresh_token",
-					}.AsUrlValues(),
-					Result:      &token,
-					Description: "oauth token",
-				}
+			// Refresh token on timer
```

Comment:
Did you just move what was in `TokenRefreshHandler` to a separate method? It is not easy to track what are the changes there.

Reply:
Not only. As I mentioned in description:
I also added back usage of LOCALIZE[30576]  and I set `config.Get().TraktToken*` values after refresh so function will not try to refresh token again (there was a bug because of those values were not set).
But everything else is a copy of old code.

BTW, If you set "diff mode = split" then it is easier to see changes in this PR. With "united" - difficult.